### DISCUSSION TITLE
How can i automate download embedded video in a webpage

### DISCUSSION THREAD

**Discussion by John4650-hub** [Skyvern-AI/skyvern#1248] [Q&A] [answered]
How can i use  skyvernAi download a video from a website that has not download button. 
I gave it a try, but skyvern-Ai wasn't able to get the video, it only played the video  for about 6 minutes then ended the session. Also the screen video recording had no sound, iam kinda new to this technology, so if i missed something try to understand.

**Comment by suchintan** [comment, marked as answer]
Hey John,

We can't download videos unless there's a download button visible on the page unfortunately.

Are you talking about youtube videos? If so I recommend checking out this project: https://github.com/ytdl-org/youtube-dl
